kivy/uix/slider.py

```diff
@@ -206,6 +213,17 @@ def on_touch_down(self, touch):
                     self.value = max(
                         self.min,
                         self.value - (self.max - self.min) / 20)
+        if self.sensitive == 'cursor':
+            for child in self.canvas.children:
+                if isinstance(child, Rectangle):
```

Comment:
My feeling are mixed here. That's the first time that we assume a certain graphical representation in order to act on the widget. And i think it's wrong.

The widget should expose the sensitive area/size, and the graphical representation (kv) must use it, or change it. But not the inverse, otherwise nobody can freely rewrite a kv rule without redoing the python code as well.